source/documentation/metrics-install-guide/Upgrading_Metrics_Store.html.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Replacing Fluentd with Rsyslog
+---
+
+# Chapter 5: Upgrading Metrics from 4.2
+
+in oVirt 4.3 Fluentd was replaced  by Rsyslog to gather metrics and logs from the oVirt hosts and engine.
+
+Rsyslog is used to collect and parse Engine, VDSM logs and Collectd metrics, add additional metadata and send the data to the remote metrics store.
+
+Note: this does not upgrade the Metrics store but the collectors.
+
+## pre-req
+
+* oVirt Metrics 4.3 with working metrics OpenShift deployment.
```

Comment:
The upgrade is for 4.3 minor versions and for 4.2 to 4.3 as well.
So the next sentence is incorrect.
Also, this is relevant for both engine and hosts.
@eslutsky

Reply:
this is not clear.. we install this on hosts only (collectors ),
 what do you mean minor?   we need to specify max version.